.eslintrc.yml

```diff
@@ -1,2 +1,8 @@
 extends:
-  - "@socialgouv/eslint-config-react"
+  - "@socialgouv/eslint-config-typescript"
+  - "plugin:import/typescript"
+parser: "@typescript-eslint/parser"
+plugins:
+  - "@typescript-eslint"
+parserOptions:
+  project: "./tsconfig.json"
```

Comment:
```suggestion
```

As the `@socialgouv/eslint-config-typescript` already set the [@typescript-eslint/parser](https://github.com/SocialGouv/linters/blob/master/packages/eslint-config-typescript/index.js#L20).
If not it means that something is missing in our config